canonical-web-and-design/ubuntu.com
# Re-enable multistage builds to reduce docker image size

QA
--

First, checkout the master branch and run:

``` bash
./run clean
./run build
time DOCKER_BUILDKIT=1 docker build -t ubuntu-com-master .
```

Now checkout this branch and run:

``` bash
time DOCKER_BUILDKIT=1 docker build -t ubuntu-com-new .
docker images | grep ubuntu-com             
```

You should see:

``` bash
ubuntu-com-new    latest 0d081e3063d8 14 seconds ago 190MB
ubuntu-com-master latest f0eca6ffccb1 4 minutes ago  259MB
```

You should also see that the second build was much quicker.

Now just check the new build actually works:

``` bash
docker run -ti -p 8001:80 ubuntu-com-new
```

Go to http://0.0.0.0:8001/advantage, try logging in, check it works.
